common/src/main/java/com/wynntils/features/user/ItemLockFeature.java

```diff
@@ -89,6 +96,12 @@ public void onInventoryClickEvent(ContainerClickEvent event) {
             return;
         }
 
+        // We want to allow interactions (except throwing) with emerald pouch even if locked
+        if (event.getClickType() != ClickType.THROW
+                && WynnItemMatchers.isEmeraldPouch(slotOptional.get().getItem())) {
+            return;
+        }
+
```

Comment:
why should emerald pouches have this exception? if someone has the "block interaction" option enabled and locks an emerald pouch, i don't see why they should expect to still use it like this.

throw actions will get blocked regardless of the setting, so this makes even less sense to me

Reply:
the reason i don't like this is that there's a very legitimate use case for blocking emerald pouches - you can essentially force close a chest by right clicking on a pouch, and a user might lock their pouch slot specifically to avoid doing that

Reply:
I will make it a setting. People complained, as far as I know, this is what legacy did